compiler/rustc_passes/src/loops.rs

```diff
@@ -64,8 +64,8 @@ impl fmt::Display for BreakContextKind {
 }
 
 #[derive(Clone)]
-struct CheckLoopVisitor<'a, 'tcx> {
-    sess: &'a Session,
+struct CheckLoopVisitor<'tcx> {
+    sess: &'tcx Session,
```

Comment:
We can get the session from the tcx, the field can be removed

Reply:
I've added a commit doing that.